webapp/pages/terms-and-conditions.vue

```diff
@@ -1,36 +1,85 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t(`site.termsAndConditions`) }}</ds-heading>
     </ds-space>
     <ds-container>
+      <no-ssr>
+        <div>
+          <ol>
+            <li v-for="section in sections" :key="section">
+              <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+              <p v-html="$t(`termsAndConditions.${section}.description`)" />
+            </li>
+          </ol>
+          <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+          <br />
+          <p>
+            <strong v-html="$t(`termsAndConditions.closing`)" />
+          </p>
+        </div>
+      </no-ssr>
       <div>
-        <ol>
-          <li v-for="section in sections" :key="section">
-            <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
-            <p v-html="$t(`termsAndConditions.${section}.description`)" />
-          </li>
-        </ol>
-        <p>{{ $t('termsAndConditions.have-fun') }}</p>
-        <br />
-        <p>
-          <strong v-html="$t('termsAndConditions.closing')" />
-        </p>
+        <ds-modal
+          v-if="isOpen"
+          v-model="isOpen"
+          v-title="$t(`site.termsAndConditionsNewTitle`)"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:54:32Z" title="Friday, August 16th 2019, 10:54:32 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,36 +1,85 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t(`site.termsAndConditions`) }}</ds-heading>
     </ds-space>
     <ds-container>
+      <no-ssr>
+        <div>
+          <ol>
+            <li v-for="section in sections" :key="section">
+              <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+              <p v-html="$t(`termsAndConditions.${section}.description`)" />
+            </li>
+          </ol>
+          <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+          <br />
+          <p>
+            <strong v-html="$t(`termsAndConditions.closing`)" />
+          </p>
+        </div>
+      </no-ssr>
       <div>
-        <ol>
-          <li v-for="section in sections" :key="section">
-            <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
-            <p v-html="$t(`termsAndConditions.${section}.description`)" />
-          </li>
-        </ol>
-        <p>{{ $t('termsAndConditions.have-fun') }}</p>
-        <br />
-        <p>
-          <strong v-html="$t('termsAndConditions.closing')" />
-        </p>
+        <ds-modal
+          v-if="isOpen"
+          v-model="isOpen"
+          v-title="$t(`site.termsAndConditionsNewTitle`)"
```


I would prefer to have a separate page to confirm the terms and conditions. You could *link* to the current terms and conditions in the same way as on the registration page. Most users don't care and just want to check the terms and conditions to proceed. Having this page would remove clutter and let users focus on the form.